infrastructure-playbooks/purge-cluster.yml

```diff
@@ -382,27 +398,29 @@
       - restart machine
       - wait for server to boot
       - remove data
-    when:
-      remove_osd_mountpoints.failed is defined
+    when: remove_osd_mountpoints.failed is defined
 
   - name: see if ceph-disk is installed
     shell: "which ceph-disk"
     failed_when: false
     register: ceph_disk_present
 
+  - name: delete dm-crypt devices if any
+    command: dmsetup remove {{ item }}
+    with_items: encrypted_partuuid.stdout_lines
```

Comment:
These two lines need to use the ``encrypted_ceph_partuuid`` variable, not ``encrypted_partuuid``.

CI is failing with:

The conditional check '{{ encrypted_partuuid.stdout_lines | length > 0 }}' failed. The error was: error while evaluating conditional ({{ encrypted_partuuid.stdout_lines | length > 0 }}): 'encrypted_partuuid' is undefined